dojson/overdo.py

```diff
@@ -62,25 +62,29 @@ def _query(self, key):
 
     def do(self, blob):
         """Translate blob values and instantiate new model instance."""
-        output = {}
+        output = []
 
         if self.index is None:
             self.build()
 
-        for key, value in iteritems(blob):
+        if isinstance(blob, GroupableOrderedDict):
+            items = blob.iteritems(repeated=True)
+        else:
+            items = iteritems(blob)
+
+        for key, value in items:
             for name, creator, field in self._query(key):
                 try:
                     data = creator(output, key, value)
-                    if getattr(creator, '__extend__', False):
```

Comment:
It's because 700 is repeatable field according to MARC21 definition so every client expects to be a list. Hence we introduced the decorator that is forcing the results to be list.